src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportFileInfo.cs

```diff
@@ -42,21 +42,21 @@ protected ExportFileInfo()
         [JsonProperty(JobRecordProperties.Sequence)]
         public int Sequence { get; private set; }
 
-        [JsonProperty(JobRecordProperties.Count)]
-        public int Count { get; private set; }
+        [JsonProperty(JobRecordProperties.TotalCount)]
+        public int TotalCount { get; private set; }
```

Comment:
@namadabu will changing these properties have any implications on the PaaS side of things?